### PR TITLE
CID-1192: improve documentation

### PR DESCRIPTION
**Pull request by vg-leanix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,12 +36,25 @@ described below.
...
+As part of the setup the vsm-broker requires a personal access token (PAT) with according rights to run effectively. For more details on how to create the PAT, see [GitHubs documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). 
+
+The following scopes are required:
+GitHub Scope  | VSM Usage
+------------- | -------------
+full `repo` including `repo:status`, `repo_deployment`, `public_repo`, `repo:invite`, `security_events`    | To read repository data for the [full scan](#broker-architecture)
+`admin:org_hook`    | To register and manage the webhook on org-level
+
 
 #### Command-line arguments
```
Comment (vg-leanix) [inline README.md:59]: @henriqamaral as per https://leanix.atlassian.net/browse/CID-1192 acceptance criteria could you pls add how to run the docker with helm?

File: README.md [hunk 2]
```diff
@@ -96,3 +141,40 @@ docker run --platform linux/amd64 \
            ...
         leanixacrpublic.azurecr.io/vsm-github-broker
 ```
+## Release Process
```
Comment (vg-leanix) [inline README.md:144]: @geoandri @henriqamaral is this accurate?

**Comment by vg-leanix** [comment]
@henriqamaral the [ticket](https://leanix.atlassian.net/browse/CID-1192) also says to put in `minimum memory requirements`? can you say something about that already?

**Review by henriqamaral — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vg-leanix) [reply README.md:59]: @henriqamaral shall i merge this PR then and you add the section on helm thereafter or do you want to use this PR to do so? either way we should do it 😆

**Comment by henriqamaral** [comment]
> @henriqamaral the [ticket](https://leanix.atlassian.net/browse/CID-1192) also says to put in `minimum memory requirements`? can you say something about that already?

Hi @vg-leanix,
The docker container will use all the memory it needs, the user can enforce the memory limit.
https://docs.docker.com/config/containers/resource_constraints/

File: README.md (hunk 1 above)
Comment (henriqamaral) [reply README.md:59]: @vg-leanix
Let's merge it. I still need to understand how helm works.

**Pull request merged**